developers/weaviate/current/configuration/authentication.md

```diff
@@ -85,19 +84,18 @@ the subject mentioned in the token.
 
 ### Configuration
 
-OpenID Connect (OIDC) can be configured like so in the respective environment in the
-`configuration.yaml`. Please see the inline-yaml comments for details around
+OpenID Connect (OIDC) can be configured like so in the respective environment variables in the Docker-Compose yaml. Please see the inline-yaml comments for details around
 the respective fields:
 
-*Note that this example uses yaml, however configuration can also be written as
-json.*
-
 ```yaml
-authentication:
-    oidc:
+services:
+  weaviate:
+    ...
+    environment:
+      ...
       # enabled (optional - defaults to false) turns OIDC auth on. All other fields in
-      # this section will only be validated if enabled is set to true.
-      enabled: true
+      # this section will only be validated if enabled is set to true.      
+      AUTHENTICATION_OIDC_ENABLED: 'true'
```

Comment:
better double check if the names of the variables should be changed

Reply:
Yup. I grabbed them from https://weaviate.io/developers/weaviate/current/installation/docker-compose.html#environment-variables. Is that what you mean?

Reply:
yes exactly :) Thanks!